### DISCUSSION TITLE
how to force global_overrides on all indices?

### DISCUSSION THREAD

**Discussion by KimkimCyber** [Security-Onion-Solutions/securityonion#12701] [Unsupported Versions]
How can I apply/force the settings of the global overrides on all the indices?
I modified the min_age and max_age for the lifecycle policy in the global overrides, but that doesn't affect the standard values of the indices.

**Comment by KimkimCyber** [comment]
Also, even when I change the values of the individual indices, it isn't reflected in Elastic itself.
Is there anything else I should do, to update the Elastic ILM from the SOC ?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/112786841/269b6791-91ea-4f7a-a659-a567f51be72e)

**Reply by cm-ops** [reply]
Did the changes take effect when the index rolled over?

**Reply by KimkimCyber** [reply]
No, I had to change it in the Dev Tools from elastic.
But normally, the values should apply after a rollover?

**Reply by cm-ops** [reply]
Once the Index moves to the next phase.

`When you apply a different policy to a managed index, the index completes the current phase using the cached definition from the previous policy. The index starts using the new policy when it moves to the next phase.` https://www.elastic.co/guide/en/elasticsearch/reference/current/update-lifecycle-policy.html
